uc_store/uc_store.install

```diff
@@ -218,6 +224,21 @@ function uc_store_update_last_removed() {
 /**
  * Implements hook_update_N().
```

Comment:
I recommend making the function message more descriptive of what the update is doing. This gets parsed and displayed for the end user. See how core does it for an example: https://github.com/backdrop/backdrop/blob/7572f3d41e157a04b262ee262884b6d463ef0ae2/core/modules/contact/contact.install#L54

Reply:
Function message for 1000 change and for 1001 added.